app/views/admin/sessions/confirm_logout.html.erb

```diff
@@ -0,0 +1,10 @@
+<h3 class="heading"><%= ts("Log Out as Admin") %></h3>
+
+<div id="logoutconfirm">
```

Comment:
Do we need wrapper elements with IDs around the confirm forms?

Reply:
I'm fairly sure we don't, this seems to be a leftover from something I was trying to debug earlier that I then forgot to get rid of. Thanks for pointing it out!